recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -540,7 +559,7 @@ def package_info(self):
         else:
             self.cpp_info.components["core"].requires.extend(["libcurl::curl", "openssl::openssl"])
 
-        if self.settings.os in ["Linux", "FreeBSD"]:
+        if self.settings.os in ["Linux", "FreeBSD", "Android"]:
             self.cpp_info.components["core"].system_libs.append("atomic")
```

Comment:
Is it possible to remove the `libatomic` dependency for architectures that don’t need it?

Reply:
yes, sorry about that!